aquamarine-vm/src/config.rs

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use fluence_faas::HostImportDescriptor;
+use std::path::PathBuf;
+
+/// Describes behaviour of the Aquamarine VM stepper.
+pub struct AquamarineVMConfig {
+    /// Path to a folder with aquamarine stepper Wasm file.
+    pub aquamarine_wasm_path: PathBuf,
```

Comment:
Should it be a path to directory containing `aquamarine.wasm`, or a path to the binary itself?
`/path/to/directory/aquamarine.wasm` or `/path/to/directory/`?